Extending/Section-Trees/tree-actions.md

```diff
@@ -84,4 +84,4 @@ Here's a list of the current User Permission codes, their alias, whether they ca
 | ®      | create               | False                      | add                 | javascript:actionNewRelationType();                                                                                                     |
 | V      | ActionRestore        | False                      |                     | This action is invoked when the content item is to be restored from the recycle bin                                                     |
 | -      | Action Null          | False                      |                     | This is used internally to assign no permissions to a node for a user and shouldn't be used in code                                     |
-| ,      | ContextMenuSeperator | False                      |                     | Used simply to define context menu separator items. This should not be used directly in any code except for creating menus              |
+| ,      | ContextMenuSeparator | False                      |                     | Used simply to define context menu separator items. This should not be used directly in any code except for creating menus              |
```

Comment:
This is unfortunately a typo in the CMS it seems 😬
https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/umbraco.cms/Actions/ContextMenuSeperator.cs#L11 

So we will have to leave it as it is until it is fixed in the CMS..